source/frontend/src/features/disposition/list/DispositionListView.test.tsx

```diff
@@ -108,7 +112,7 @@ describe('Disposition List View', () => {
   it('searches by pid', async () => {
     let results = mockPagedResults([
       {
-        ...mockDispositionFileResponse(),
+        ...(mockDispositionFileResponse() as unknown as Api_DispositionFile),
```

Comment:
I'm having trouble understanding the need for this, if the entire point of this mock is to create a disposition file, why are you needing to cast this to an Api_DispositionFile via an unknown?

Reply:
I am casting from the new generated models ApiGen_Concepts_XXX I am limiting the use of the new models scoped to what I am currently working and not trying to spill over everywhere.

Reply:
ah ok. I see. That is fine for now then.

Reply:
Actually, can you log a follow-up task for is74 that would cover correcting this once we merge 5.1

That way we don't forget.

Reply:
Task created.